File: Lintcode/1-A+B_Problem/answer.java
public class Solution {
    /**
     * @param a: An integer
     * @param b: An integer
     * @return: The sum of a and b
     */
    public int aplusb(int a, int b) {
        // write your code here
        int carry;
        while(b != 0){
            carry = a & b;
            a = a ^ b;
            b = carry << 1;
        }
        
        return a;
    }
}

File: Lintcode/172-Remove_Element/answer.py
class Solution:
    """
    @param: A: A list of integers
    @param: elem: An integer
    @return: The new length after remove
    """
    def removeElement(self, A, elem):
        # write your code here
        
        size = len(A)
        indexes = []
        
        for i in range(size):
            if A[i] == elem:
                indexes.append(i)
                
        for i in sorted(indexes, reverse = True):
            del A[i]
            
        return A
